jira/jira-v1.asciidoc

```diff
@@ -0,0 +1,104 @@
+|Child Request |/rest/api/2/issue/${LW_PARENT_DATA_KEY}/worklog | GET |startAt=0&maxResult=50 |Returns all worklogs. The request requires the 'issue Id'. Internally, the plugin replaces the variable ${LW_PARENT_DATA_KEY} with the 'issue id', which value is extracted from the 'issue object' (root object) by setting the parentDataKey property.
+|Child Request |/rest/api/2/attachment/content/${LW_PARENT_DATA_KEY}|GET|-|Use root response containing attachment Id's to retrieve attachments. The following jsonpath is used to retrieve the attachment id's `fields.attachment[*]` 
+|=======================
+
+
+== Terminology
+
+The following terms are provided as a reference.
+
+[options="header",cols="1s,1"]
+|=======================
+
+|Term|Description
+|Service Endpoints|The list of service endpoints from which the data is retrieved. Each service endpoint configures a root endpoint request.
+|Root Request|The type of request to retrieve a list of root data objects.
+|Child Request|The type of request to retrieve additional information for the root data objects. The child requests will be performed per each root data object.
+|Root Response Mapping|Defines the mapping between the response and data objects to be indexed.
+|Child Response Mapping |Defines the mapping between the child response and child data objects to be indexed.
+|Data Path|The path to access a specific data object within a response. For example, to access a list of elements named with key `objects`, the DataPath would be `objects`. If not provided, the entire response body will be indexed. This property accepts JsonPath expressions e.g.objects, objects[*] or $.objects[*]
+|DATA ID|The identifier key for the data objects extracted with 'Data Path', this value will be used to build the solr-document's ID. If not provided, a random UUID will be used. This property accepts JsonPath expressions.
+|Parent Data Key|Key to extract data from the root/parent response used in the subsequent request. The extracted value is used to replace the ${LW_PARENT_DATA_KEY} variable in the child request configuration (endpoint, query params or body). For example, endpoint: /api/path/${LW_PARENT_DATA_KEY}/additionalInfo.
+|Child Data Path|The path to access a specific object within a child response. For example, to access a list of elements named with the key `objects`, the ChildDataPath would be `objects`. If not provided, the entire response body will be indexed.
+|Child Data ID|The identifier key for the child data object, where the value is the solr-document's ID. Enter this when the `Custom Solr Field` is empty, otherwise the solr-document's ID will be a random universally unique identifier (UUID).
```

Comment:
The identifier key for the child data object, which value will be the solr-document's ID. Fill this when 'Custom Solr Field' is empty, otherwise the solr-document's ID will be a random UUID.

Reply:
Good catch.  I think we should set it to self. Thoughs?